src/app.js

```diff
@@ -88,13 +95,33 @@ app.get("/company/:companyId", (req, res) => {
 
 app.get("/company", (req, res) => {
   // get query out from req
+  if (!req.query.offset) {
+    res.status(400);
+    res.send("/company request need a query for offset!");
+    return;
+  }
+  const offset = req.query.offset;
+  const limit = req.query.limit ?? 5;
```

Comment:
we should not let the user decides `limit`, we should hard coded in our backend.
imagine a doggy user set limit to 1 mil and then your server will hang